TESTING.md

```diff
@@ -2,60 +2,198 @@
 To run tests in go-vcloud-director, users must use a yaml file specifying information about the users vcd. Users can set the `VCLOUD_CONFIG` environmental variable with the path.
 
 ```
-export VCLOUD_CONFIG = $HOME/test.yaml
+export VCLOUD_CONFIG=$HOME/test.yaml
 ```
 
 If no environmental variable is set it will default to $HOME/config.yaml.
 
+## Example Config file
 
-# Example Config file
-
-```
+```yaml
 provider:
-  user: root
```

Comment:
I have tried with administrator account and the tests failed with a cryptic error message. I was able to run them after changing to _Org_ administrator. I suggest changing the sample from "root" to "orgAdmin".

Granted, we want the tests to create all of the below items automatically. As soon as we start creating Orgs and vDCs, we will require the System administrator.

Reply:
yes for that we will add an org variable into the provider as well, so that when the tests authenticate it does so with the provider org instead of the vcd org used for testing.